crates/ruma-client-api/src/discovery/discover_homeserver.rs

```diff
@@ -39,7 +39,11 @@ ruma_api! {
 
         /// Information about the authentication server to connect to when using OpenID Connect.
         #[cfg(feature = "unstable-msc2965")]
-        #[serde(rename = "m.authentication", skip_serializing_if = "Option::is_none")]
+        #[serde(
+            rename = "org.matrix.msc2965.authentication",
+            alias = "m.authentication",
```

Comment:
Don't we want to add that alias only after things become stable?

In other words, once the `#[cfg(feature = "unstable-msc2965")]` gets removed, the alias gets added. Or rather the `org.matrix.msc2965.authentication` becomes the alias.

Reply:
There are other places where we do that already. The idea is that clients don't need to wait for a Ruma update to be able to receive the stable prefix.